server/lib/nl/utils.py

```diff
@@ -525,28 +525,26 @@ def get_immediate_parent_places(
     main_place_dcid: str, parent_place_type: str,
     counters: ctr.Counters) -> List[detection.Place]:
   start = time.time()
-  resp = dc.property_values_v1([main_place_dcid], 'containedInPlace')
+  resp = util.property_values([main_place_dcid], 'containedInPlace')
   counters.timeit('get_immediate_parent_places', start)
   results = []
-  for item in resp.get('data', []):
-    if item.get('node', '') != main_place_dcid:
+  for value in resp['data'][main_place_dcid].get('arcs', {}).get(
```

Comment:
does resp have a 'data' field & an 'arcs' field? Because doesn't util.property_values just return a dict of dcid to property values?

Reply:
Good catch! This one should use the raw response as it fetches name and type. This also explains the NL test failure.